variant-caller-wdl-checker/topmed_freeze8_caller_checker.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checkerWorkflow.input_cram_files": [
```

Comment:
Doesn't Terra have issues with https? Would it be good to have a file with the gs versions as well, especially since we're now trying to get them to import JSON files from Dockstore as well as the workflow itself.

Reply:
Added a  GCS JSON input file